coretex/entities/task_run/task_run.py

```diff
@@ -335,6 +335,78 @@ def submitMetrics(self, metricValues: Dict[str, Tuple[float, float]]) -> bool:
         response = networkManager.post(f"{self._endpoint()}/metrics", parameters)
         return not response.hasFailed()
 
+    def submitOutput(self, parameterName: str, value: Any) -> None:
+        """
+            Submit an output of this task to act as a parameter in tasks
+            downstream in the Workflow
+
+            Parameters
+            ----------
+            parameterName : str
+                name of the parameter
+            value : Any
+                value that will be sent (id of the object will be sent
+                in case Coretex objects like CustomDataset, Model etc. is submited)
+
+            Example
+            -------
+            >>> from coretex import TaskRun
+            \b
+            >>> result = ExecutingTaskRun.current().submitOutputParameter(
+                    parameterName = "outputDataset"
+                    value = outputDataset
+                )
+            >>> print(result)
+            True
```

Comment:
`result` is no longer returned, instead a `NetworkRequestError ` exception is raised. Update this in both `submitOutput` and `submitOutputs` methods